db_stress_tool/db_stress_test_base.cc

```diff
@@ -729,6 +729,14 @@ void StressTest::OperateDb(ThreadState* thread) {
 
       MaybeClearOneColumnFamily(thread);
 
+      if (thread->rand.OneInOpt(FLAGS_manual_wal_flush_one_in)) {
+        Status s = db_->FlushWAL(false /* sync */);
```

Comment:
sync=true is important. Can we randomly pick the value for `sync` since we don't have any verification depending on it anyways?